docs/gcloud-auth.rst

```diff
@@ -58,15 +58,66 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful
```

Comment:
I'm not saying it's about 80 chars, but I am saying I have a hard time grokking what semantic line breaks are / carrying out the process of using them.

From where I'm sitting, it ain't broke, what are we fixing?